src/support/SupportResourceList.ts

```diff
@@ -0,0 +1,84 @@
+import _ from 'lodash/fp';
+import { div, h } from 'react-hyperscript-helpers';
+import * as Style from 'src/libs/style';
+import * as Utils from 'src/libs/utils';
+import { SupportResourceListItem, SupportResourceListItemProps } from 'src/support/SupportResourceListItem';
+import { supportResources, SupportResourceType } from 'src/support/SupportResourceType';
+
+interface SupportResourceListProps {
+  queryParams: {
+    selectedType: string | undefined;
+    resourceName: string | undefined;
+  };
+}
+
+export const SupportResourceList = (props: SupportResourceListProps) => {
+  const selectedType = props.queryParams.selectedType || '';
+  const resourceName = props.queryParams.resourceName || '';
+  const supportResourceListWidth = 350;
+
+  const makeResourceListItemProps = (resourceType: SupportResourceType): SupportResourceListItemProps => {
+    return {
+      resourceType,
+      isActive: !!selectedType && resourceType.resourceType === selectedType,
+    };
+  };
+
+  return div({ role: 'main', style: { display: 'flex', flex: 1, height: `calc(100% - ${Style.topBarHeight}px)` } }, [
+    div(
+      {
+        style: {
+          minWidth: supportResourceListWidth,
+          maxWidth: supportResourceListWidth,
+          boxShadow: '0 2px 5px 0 rgba(0,0,0,0.25)',
+          overflowY: 'auto',
+        },
+      },
+      [
+        div({ role: 'list' }, [
```

Comment:
Is it possible to use semantic elements (`ul` and `li`) instead of `div`s with roles here?

Reply:
I just copied what the billing list page does. I tried using `ul` and `li` and it added bullets which I did not like.

Reply:
Fair enough. For what it's worth, the default list styling can be overridden with CSS `style={{ listStyle: 'none', padding: 0 }}`.